reports/templates/report.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% include staticfiles %}
+
+{% block title %}{{ month_name }} {{ year }} invoice for {{ organization.name }}{% endblock title %}
+
+{% block content %}
+  <div class="page-title text-xs-center color-white">
+    <h1 class="page-title__text">Invoice</h1>
+  </div>
+
+  <div class="row small-collapse page-content clearfix">
+    <div class="page-content__inner small-12 medium-10 medium-offset-1 columns background-white">
+      <div class="page-content__header small-10 small-offset-1 columns">
+        <div class="box-alert background-cyan-very-very-dark-desaturated color-white">
+          <p class="box-alert__content text-center">
+            Invoice for {{ month_name }} {{ year }} for {{ organization.name }}
+          </p>
+        </div>
+        {% for fork, fork_data in billing_data.items %}
+          <div>
+            <strong>For watched fork:</strong> {{ fork.fork }}
+          </div>
+          <table style="table-layout: fixed; width: 100%;">
+            <thead>
+            <tr>
+              <th></th>
+              <th>First billed day</th>
+              <th>Last billed day</th>
+              <th>Days</th>
+              <th>Charge</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for instance, appservers in fork_data.instances.items %}
+              <tr>
+                <td style="text-align: left;">{{ instance }}</td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+              </tr>
+              {% for appserver in appservers %}
```

Comment:
Appservers with 0 days (e.g. because they're much newer than the report's month) will be shown as 0€, but it's informational and for now we can remove it manually from the invoice if it makes it too long. We could add an `if` here later if we want to disable them.